test/src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -444,6 +452,10 @@ public void after() throws Exception {
             clients.clear();
 
         } finally {
+            // Make sure the unit test flag gets reset in case a test turned
```

Comment:
> reset in the context of a boolean field is normally understood to mean set to false

Really? I'd usually think of it to mean "set back to its default/initial value" (which in this case is `true`), but would interpret it in the context of the code it's connected to.

I think it's perfectly fine having it here but will remove and do it `@After` in `StartupUtilTest`.